packages/boa/esm/loader.mjs

```diff
@@ -0,0 +1,39 @@
+import Boa from '../lib/index.js';
```

Comment:
```suggestion
import Boa from '../';
```

Reply:
Doesn't seem to automatically look for `package.json`.

Reply:
Weird, we have defined the `main` as "lib/index.js" at https://github.com/alibaba/pipcook/blob/master/packages/boa/package.json#L5.